frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/review-and-create.tsx

```diff
@@ -57,11 +60,22 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
           <Trans t={t} ns="ceph-storage-plugin" values={osdSize}>
             Requested Cluster Capacity:&nbsp;
             <span className="text-secondary">
-              {{ osdSize }} TiB&nbsp;
+              {osdSize.toString()} TiB&nbsp;
```

Comment:
Is this necessary ?

Reply:
Yes, it's wasn't working. It was saying `TiB * 3 = 6TiB`  the number was not present